Makefile

```diff
@@ -105,7 +105,9 @@ $(KODATA):
 
 .PHONY: apply
 apply: $(KO) $(KODATA) ## Deploy the controller into the current kubernetes cluster.
-	helm upgrade --install dev charts/aws-node-termination-handler-2 --namespace nthv2 --create-namespace \
+	helm upgrade --install dev charts/aws-node-termination-handler-2 \
+		--namespace ${CLUSTER_NAMESPACE} \
```

Comment:
The `delete` target still has hard-coded namespace. Should be $CLUSTER_NAMESPACE?